src/pages/EditMonitor.vue

```diff
@@ -144,8 +147,26 @@
                                 </div>
                             </div>
 
-                            <!-- Json Query -->
-                            <div v-if="monitor.type === 'json-query'" class="my-3">
+                            <!-- NUT -->
+                            <!-- For NUT Type -->
+                            <template v-if="monitor.type === 'nut'">
+                                <div class="my-3">
+                                    <label for="upsName" class="form-label">UPS {{ $t("Name") }}</label>
```

Comment:
I think adding a description on where to obtain the correct ups name might be worth it to prevent other users spending excessive time on this field

Reply:
OK I agree, do you think it makes sense to add it on the field description? If it's too long, where would that sort of documentation go?

Screenshot as an example of where it could live:
<img width="588" alt="image" src="https://github.com/louislam/uptime-kuma/assets/5633265/3fe1695a-734f-431f-b43f-b7c9afab4461">